bestbook/api/books.py

```diff
@@ -64,105 +64,104 @@ class BookGraph(core.Base):
     __tablename__ = "bookgraph"
     __table_args__ = (
         PrimaryKeyConstraint(
-            'submitter', 'winner_work_olid', 'competitor_work_olid', 'topic_id'
+            'submitter', 'winner_work_olid', 'contender_work_olid', 'topic_id'
         ),
         UniqueConstraint(
-            'submitter', 'winner_work_olid', 'competitor_work_olid', 'topic_id',
+            'submitter', 'winner_work_olid', 'contender_work_olid', 'topic_id',
             name='_book_edge_uc'
         )
     )
 
     submitter = Column(Unicode, nullable=False) # e.g. @cdrini - Open Library username
     winner_work_olid = Column(Unicode, nullable=False) # Open Library ID (required)
-    # rename to contender_work_olid
-    competitor_work_olid = Column(Unicode, nullable=False) # Open Library ID (required)
+    contender_work_olid = Column(Unicode, nullable=False) # Open Library ID (required)
     topic_id = Column(Integer, ForeignKey("topics.id")) # TBBO what?
-    # review_id = Column(BigInteger, ForeignKey("reviews.id"))
-    annotation = Column(Unicode) # Free-form answer  # XXX rm
+    review_id = Column(BigInteger, ForeignKey("reviews.id"))
```

Comment:
Note:
In a situation like 3 tournament_graph entries: `(A>B, A>C, A>D) w/ Review.id .... On Topic.id`, TBBO is sum(winner), for each winner unique(review.id)

Reply:
Note:
`recommendations` table should specify winner per person on Topic.id